dot-studio/golang

```diff
@@ -42,19 +42,18 @@ function setup_go_workspace() {
   # Currently the scaffolding-go has all the logic to setup the
   # Go Workspace, lets leverage all its functionality
   echo "=> Configuring Go Workspace. [please wait]"
-  export scaffolding_go_gopath=${scaffolding_go_gopath:-/hab/cache/src/go}
+  export scaffolding_go_gopath="${scaffolding_go_gopath:-/hab/cache/src/go}"
   install "$pkg_scaffolding" >/dev/null
   source "$(hab pkg path "$pkg_scaffolding")/lib/"*
   export PATH="$PATH:${scaffolding_go_gopath}/bin"
 
   # Verify if we have the go_before callback
   # If not lets setup the workspace ourself
   go_before_callback=scaffolding_go_before
-  if [ "$(type -t $go_before_callback)" = function ]
-  then
+  if [[ "$(type -t $go_before_callback)" = function ]]; then
     eval "$go_before_callback"
   else
-    mkdir -p "${scaffolding_go_workspace_src}/${scaffolding_go_base_path}"
+    mkdir -p "${scaffolding_go_workspace_src}/${scaffolding_go_base_path}" || exit
```

Comment:
I'm not sure this `exit` is a good idea. I think this will exit the entire studio. Why do we want this?

Reply:
No, you're right. that's a good catch